### ISSUE TITLE
Conditional variables don't work for {% extends %} tag when Blitz Hints utility is enabled

### ISSUE THREAD

**Issue by nikolenko-dmitriy** [putyourlightson/craft-blitz#632]
### Bug Report

**Tested in 2 different cases:**

1. Setting the custom variable to define the correct layout to extend. Below is a simple example:
```
{% set isJson = true %}

{% extends isJson ? '_layouts/ajax' : '_layouts/content' %}
```
Page fails with error `Variable "isJson" does not exist.`

2. Using the default craft global variable `currentUser`:
```
{% extends not currentUser ? '_layouts/content' : '_layouts/content_with_sidebar' %}
```
Page fails with error `Variable "currentUser" does not exist.`

**Note:** the same examples work fine when Blitz setting `hintsEnabled` set to `false`.

**Website uses some other important plugins:**
- Craft Commerce v 4.5.2
- Sprig v 2.7.3
- Blitz v 4.13.0. Also tested with v 4.12.0, v 4.11.0, v 4.10.0 and v 4.9.3, the result is the same

### Plugin Version

4.13.0

### Craft CMS Version

4.8.1

### PHP Version

8.1.27

**Comment by bencroker** [comment]
I’m unable to replicate this using Blitz 4.13.0 on Craft 4.8.1. Can you show me the full stack trace of the error?

**Comment by nikolenko-dmitriy** [comment]
> I’m unable to replicate this using Blitz 4.13.0 on Craft 4.8.1. Can you show me the full stack trace of the error?

Here it is [bugreport.txt](https://github.com/putyourlightson/craft-blitz/files/14569442/bugreport.txt)

**Comment by bencroker** [comment]
Thanks! I wonder if this is fixed by https://github.com/putyourlightson/craft-blitz-hints/commit/30cba8eee510ae75325ffb77c0619a36265691c0. 
Just released in Blitz Hints [1.2.2](https://github.com/putyourlightson/craft-blitz-hints/releases/tag/1.2.2), can you please run `composer update` to get the latest version and test?

**Comment by bencroker** [comment]
Confirmed the fix and released in Blitz [4.13.1](https://github.com/putyourlightson/craft-blitz/releases/tag/4.13.1) too, just to be safe. Thanks again for the bug report!

**Comment by nikolenko-dmitriy** [comment]
> Confirmed the fix and released in Blitz [4.13.1](https://github.com/putyourlightson/craft-blitz/releases/tag/4.13.1) too, just to be safe. Thanks again for the bug report!

Yes, the fix works for me as well. Thank you!
